package.json

```diff
@@ -5,9 +5,7 @@
   "scripts": {
     "spellcheck": "mdspell --en-gb --report --ignore-acronyms --ignore-numbers --no-suggestions 'content/**/*.md'",
     "broken-link-local": "blcl --recursive --ordered --exclude-external ./public/",
-    "broken-link-all": "blcl --recursive --ordered ./public/",
-    "test": "npm run spellcheck && npm run broken-link-local",
```

Comment:
Could we keep these scripts, and just copy the diff from https://github.com/sourcebots/volunteer-docs/pull/36/. It means that (if required), we can still run the full link checker locally

Reply:
Why? They don't work (specifically, once #31 merges, `broken-link-all` will _always_ get stuck on `uos.svg`) and I'd rather not leave broken things in place.
If we want to fix them, they're in the history and easy to get back.

Reply:
true, true